database/index.js

```diff
@@ -57,6 +57,47 @@ var updateOrder = (pic, order, cb, indx) => {
     });
 };
 
```

Comment:
Love how you implemented the function for your CRUD operations. Great work here.